.github/CODEOWNERS

```diff
@@ -88,6 +87,7 @@
 /functions                       @GoogleCloudPlatform/java-samples-reviewers @yoshi-approver @GoogleCloudPlatform/torus-dpe @GoogleCloudPlatform/functions-framework-google
 # ---* Fully Eng Owned
 /appengine-*                     @GoogleCloudPlatform/java-samples-reviewers @yoshi-approver @GoogleCloudPlatform/serverless-runtimes
+/asset                           @GoogleCloudPlatform/java-samples-reviewers @yoshi-approver @GoogleCloudPlatform/cloud-asset-analysis-team @GoogleCloudPlatform/cloud-asset-platform-team
```

Comment:
@averikitsch Before approving please grant write access to this repo to new teams: cloud-asset-analysis-team, cloud-asset-platform-team

Reply:
Access granted. This is ready for merging.